multiselect.module

```diff
@@ -70,39 +70,51 @@ function multiselect_theme() {
 function template_preprocess_multiselect(&$variables) {
   $element = $variables['element'];
   Element::setAttributes($element, array('id', 'name', 'size', 'required'));
-  //_form_set_attributes($element, array('form-multiselect'));
+
+  $available_options = multiselect_get_options('available', $element);
```

Comment:
In Drupal 8, this may be best done through a service or something... I'll take a closer look